platformio_tubes.ini

```diff
@@ -147,7 +147,6 @@ build_unflags = ${env:esp32-c3-athom.build_unflags}
 build_flags =
   ${tubes_no_mic.build_flags}
   -D WLED_WATCHDOG_TIMEOUT=0
-;  -D LOLIN_WIFI_FIX ; seems to work much better with this
```

Comment:
this just set the power setting.  This can now be set via the WLED_WIFI_POWER_SETTING macro